### ISSUE TITLE
Userland: Give all applications a decent manpage + a link in the Help menu

### ISSUE THREAD

**Issue by cubiclove** [SerenityOS/serenity#21091]
A fair number of applications, demos and games are missing a manpage, have just a stub or are missing a link to the manual in the `Help` menu.

Applications should have:
- An initial or updated manpage
- A link to this in the `Help` menu (unless it's an application's settings app)

### Applications:

- [x] 2048
- [ ] 3DFileViewer
    - [x] New/updated/OK manpage
    - [ ] Link in Help/context menu
- [ ] About
    - Explain the revision number
    - Include links to the website for more information
- [ ] AnalogClock
- [ ] Assistant
    - [x] New/updated/OK manpage
    - [ ] Link in Help/context menu
- [x] BrickGame
- [ ] Browser
- [ ] BrowserSettings
- [ ] Calculator
    - [x] New/updated/OK manpage
    - [ ] Link in Help/context menu
- [ ] Calendar
- [ ] CalendarSettings
- [ ] CatDog
    - [x] New/updated/OK manpage
    - [ ] Link in Help/context menu
- [x] CertificateSettings
- [x] CharacterMap
- [ ] Chess
    - [ ] New/updated/OK manpage
    - [x] Link in Help/context menu
- [ ] ClockSettings
- [ ] ColorLines
    - [ ] New/updated/OK manpage
    - [x] Link in Help/context menu
- [ ] DisplaySettings
- [ ] Eyes
    - Could do with a short explanation + example
    - [ ] New/updated/OK manpage
    - [x] Link in Help/context menu
- [ ] FileManager
- [ ] FlappyBug
    - [ ] New/updated/OK manpage
    - [x] Link in Help/context menu
- [x] Flood
- [ ] FontEditor
    - Could do with a grammar check
    - [ ] New/updated/OK manpage
    - [x] Link in Help/context menu
- [x] GameOfLife
    - Add wikipedia link to Conway's Game of Life?
- [ ] GamesSettings
- [x] GMLPlayground
- [ ] HackStudio
- [x] Hearts
- [x] Help
- [x] HexEditor
- [x] ImageViewer
- [ ] KeyboardMapper
- [ ] KeyboardSettings
- [ ] LibGfxDemo & LibGfxScaleDemo
- [x] Magnifier
- [ ] Mail
- [ ] MailSettings
- [ ] Mandelbrot
- [ ] Maps
- [x] MasterWord
- [x] Minesweeper
- [ ] ModelGallery
- [x] MouseSettings
- [ ] NetworkSettings
- [ ] PartitionEditor
- [ ] PDFViewer
- [ ] Piano
- [ ] PixelPaint
    - Will be submitting soon
- [ ] Presenter
    - [x] New/updated/OK manpage
    - [ ] Link in Help/context menu
- [x] Profiler
- [ ] Run
- [ ] Screensaver (inc. Gradient, Starfield & Tubes)
- [ ] Settings
- [ ] Snake
    - [ ] New/updated/OK manpage
    - [x] Link in Help/context menu
- [ ] Solitaire
    - Reorganize Help menu
    - [x] New/updated/OK manpage
    - [x] Link in Help/context menu
- [ ] SoundPlayer
- [ ] SpaceAnalyzer
- [ ] Spider
    - [x] New/updated/OK manpage
    - [ ] Link in Help/context menu
- [ ] Spreadsheet
- [x] SQLStudio
- [ ] SystemMonitor
- [x] Terminal
- [ ] TerminalSettings
- [x] TextEditor
- [ ] ThemeEditor
- [ ] VideoPlayer
- [ ] WidgetGallery

I will gradually be working through these

**Comment by ADKaster** [comment]
Seems this list has quite a lot of overlap with https://github.com/SerenityOS/serenity/issues/11311

**Comment by cubiclove** [comment]
@ADKaster You're right. This is slightly expanded in that it includes expanding and improving manpages (some have been stubs or had pretty poor grammar) and making sure they are linked in the application's Help Menu.

Would a better solution be to ask Sam for permission to edit the existing issue as I'll make sure to keep it up to date?

**Comment by AtkinsSJ** [comment]
Unfortunately I can't give you permission to edit that issue. GitHub either lets you just edit your own, or EVERYTHING.

I've no idea how up to date the other issue is, so maybe it'd be best to make that just about Utilities now, and you can keep an eye on Applications, since you already seem to have a good idea of what the situation is. :^)

Thanks for taking these on by the way!

**Comment by Popaulol** [comment]
For the following applications, adding a manual link to the help-context menu, doesn't really make sense since they either don't use normal windows, or are meant to only temporarily show when used. 
For those marked with * a help button somewhere would still make sense but adding it as a Context Menu wouldn't make much sense

- Assistant *
- CatDog
- Run *

**Comment by Popaulol** [comment]
So I have done a bit of Work for this in #21731

**Comment by cubiclove** [comment]
Thanks for your help @Popaulol!

PSA: I haven’t forgotten about this task, I am working on the Pixel Paint manpage which is going to take some time to get right.

**Comment by kanak227** [comment]
Hey @cubiclove Just wanted to know if this issue is still open. If yes , Can I be assigned to work on the same?

**Comment by Popaulol** [comment]
> Hey [@cubiclove](https://github.com/cubiclove) Just wanted to know if this issue is still open. If yes , Can I be assigned to work on the same?

Hi @kanak227 On this repo issues aren't usually (or really ever) assigned to people, especially for larger issues like this one.

You can just work on the Issue and then reference the issue in the PR you're submitting.
